Gruntfile.js

```diff
@@ -0,0 +1,30 @@
+/*
+	Copyright 2020 OCAD University.
+
+	Licensed under the BSD-3-Clause license.
+*/
+
+"use strict";
+
+module.exports = function (grunt) {
+	grunt.config.init({
+		lintAll: {
+			sources: {
+				md: [ "*.md", "!.github/*.md"],
```

Comment:
Is there a reason to exclude the markdown files from the .github directory?

Reply:
Sorry, forgot to reply to your question. No I don't think we've excluded it from other projects, but most don't include markdown in the .github directory.